src/js/components/ServiceDetailConfigurationTab.js

```diff
@@ -56,8 +57,8 @@ class ServiceDetailConfigurationTab extends React.Component {
       this.props.service.getVersions().get(this.state.selectedVersionID);
 
     MarathonStore.editService(
-      ServiceUtil.getAppDefinitionFromService(
-        new Service(serviceConfiguration)
+      ServiceUtil.getDefinitionFromSpec(
```

Comment:
Yup, I kept it for making the PR functional on it's own until we have that PR merged.